jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -479,13 +493,24 @@ public JaegerSpan startManual() {
     private ScopeManager scopeManager = new ThreadLocalScopeManager();
     private BaggageRestrictionManager baggageRestrictionManager = new DefaultBaggageRestrictionManager();
     private boolean expandExceptionLogs;
+    private JaegerObjectFactory objectFactory;
```

Comment:
`protected final` ?

Reply:
Maybe. `final` is fine. This might only be in C++, but I've heard to avoid protected fields. The logic is that you might want to remove the field later and will definitely break subclasses. If you use a protected setter/getter, then you can write functions to maintain some sort of compatibility.